src/main/webapp/dataverseuser.xhtml

```diff
@@ -571,7 +571,7 @@
                             <div class="form-horizontal">
                                 <div class="form-group" jsf:rendered="#{DataverseUserPage.editMode == 'CREATE' or DataverseUserPage.editMode == 'EDIT'}">
                                 <!-- Username -->
-                                    <label for="userNameEmail" class="col-sm-3 control-label">
+                                    <label for="dataverseUserForm:accountInfoView:userName" class="col-sm-3 control-label">
```

Comment:
@sekmiller rather than add the whole name (though dropping the mail was definitely needed), did you try using <h:outputLabel> or <p:outputLabel>?  If that works, that would be better, in case the structure changes some day.

Reply:
I didn't try it originally because I wasn't sure if I could replicate the formatting. I'll see if I can get it to look right with outputLabel.